File: python/674.py
class Solution:
    def findLengthOfLCIS(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """

        ans = 1
        temp = 1
        l = len(nums)
        if l == 0:
            return 0
        if l == 1:
            return 1
        for i in range(1,l):
            if nums[i] > nums[i - 1]:
                temp += 1
            else:
                ans = max(ans,temp)
                temp = 1
        ans = max(ans,temp)
        return ans

File: java/674.java
class Solution {
    public int findLengthOfLCIS(int[] nums) {
        int ans = 1,l = nums.length;
        int tmp = 1;
        for(int i = 0;i <l-1;i++){
            if(nums[i] < nums[i+1]){
                tmp ++;
            }else{
                tmp = 1;
            }
            ans = Math.max(tmp,ans);
        }
        return Math.min(ans,l);
    }
}